src/parsers/manifest/dash/common/indexes/template.ts

```diff
@@ -487,7 +498,6 @@ export default class TemplateRepresentationIndex implements IRepresentationIndex
    */
   _replace(newIndex : TemplateRepresentationIndex) : void {
     this._index = newIndex._index;
-    this._aggressiveMode = newIndex._aggressiveMode;
```

Comment:
I have seen previously in the migration guide from v3 to v4 that `transportOptions.agressiveMode` has been removed.
Is `aggresiveMode` still a thing in v4 ?

Reply:
No it is not, but here we're merging in v3.

`aggressiveMode` isn't really useful anyway